src/lib/rest/StringFilter.cpp

```diff
@@ -441,6 +442,15 @@ bool StringFilterItem::valueGet
     *stringP = s;
   }
 
+  if (*valueTypeP == SfvtString)  // FIXME PR: perhaps not calling forbiddenChars if pattern matching ...
```

Comment:
Not before discussing it first.

It seems to be we need to exclude the RHS of pattern matching from the forbiddenChars check.
To do that, we will need the OP of the string filter down in StringFilterItem::valueGet.